src/api/cloud/project/io/stream/subscription/cloud-project-io-stream-subscription.v6.service.js

```diff
@@ -0,0 +1,21 @@
+angular
+  .module('ovh-api-services')
+  .service('OvhApiCloudProjectIoStreamSubscriptionV6', ($resource) => {
+    const resource = $resource('/cloud/project/:serviceName/io/stream/:streamId/subscription/:subscriptionId', {
+      serviceName: '@serviceName',
+      streamId: '@streamId',
+      subscriptionId: '@subscriptionId',
+    }, {
+      resetCursor: {
+        method: 'POST',
+        url: '/cloud/project/:serviceName/io/stream/:streamId/subscription/:subscriptionId/resetCursor',
+      },
+      getStats: {
+        url: '/cloud/project/:serviceName/io/stream/:streamId/subscription/:subscriptionId/stats',
```

Comment:
No cache ?

Reply:
There is no need to handle the cache at this level.